portfolio/karma.conf.js

```diff
@@ -0,0 +1,70 @@
+// Karma configuration
+// Generated on Fri Jul 17 2020 10:17:57 GMT+0000 (Coordinated Universal Time)
+
```

Comment:
This is the same code as in #5 

Did you put one branch on top of another? Before #5 is merged, this will be hard to review :(

Reply:
Yes I wasn't sure exactly what the protocol is here when I have several stages of work that I had finished, whether to submit one PR at a time? Or to submit both?

Reply:
Ideally, different work wouldn't depend on each other. And if they do... I don't have a good answer.

Sometimes it's possible to split the common parts to a separate PR and submit it first. Also it would help to not add too many commits, but when fixing code added in previous commit, amend that commit instead (unless it's already merged master branch, of course); then it becomes easier to review commits one by one.

Reply:
I can see how lots of commits definitely make it harder to navigate and review, I'll do some more practice on rebasing and ammending commits so I can use that next time instead when needed. Thanks